include/cupla/cudaToCupla/runtime.hpp

```diff
@@ -77,3 +77,11 @@
 #define make_cudaPos(...) make_cuplaPos(__VA_ARGS__)
 
 #define make_cudaPitchedPtr(...) make_cuplaPitchedPtr(__VA_ARGS__)
+//math
+#define __fdividef(a,b) ((a)/(b))
```

Comment:
please create a own pull request for this changes because they are independent of the example and extent the core part of **cupla**